components/jira/actions/update-issue/update-issue.mjs

```diff
@@ -1,171 +1,105 @@
-// legacy_hash_id: a_3Li68z
-import { axios } from "@pipedream/platform";
+import jira from "../../jira.app.mjs";
 
 export default {
   key: "jira-update-issue",
   name: "Update Issue",
-  description: "Updates an issue. A transition may be applied and issue properties updated as part of the edit.",
-  version: "0.1.1",
+  description: "Updates an issue. A transition may be applied and issue properties updated as part of the edit, [See the docs](https://developer.atlassian.com/cloud/jira/platform/rest/v3/api-group-issues/#api-rest-api-3-issue-issueidorkey-put)",
+  version: "0.1.2",
   type: "action",
   props: {
-    jira: {
-      type: "app",
-      app: "jira",
-    },
+    jira,
     issueIdOrKey: {
-      type: "string",
-      description: "The ID or key of the issue to update.",
+      propDefinition: [
+        jira,
+        "issueIdOrKey",
+      ],
     },
     notifyUsers: {
       type: "boolean",
+      label: "Notify users",
       description: "Whether a notification email about the issue update is sent to all watchers. To disable the notification, administer Jira or administer project permissions are required. If the user doesn't have the necessary permission the request is ignored.",
       optional: true,
     },
     overrideScreenSecurity: {
       type: "boolean",
+      label: "Override screen security",
       description: "Whether screen security should be overridden to enable hidden fields to be edited. Available to Connect app users with admin permissions.",
       optional: true,
     },
     overrideEditableFlag: {
       type: "boolean",
+      label: "Override editable flag",
       description: "Whether screen security should be overridden to enable uneditable fields to be edited. Available to Connect app users with admin permissions.",
       optional: true,
     },
     transition: {
       type: "object",
-      description: "Details of a transition. Required when performing a transition, optional when creating or editing an issue.",
+      label: "Transition",
```

Comment:
How about setting this as a `propDefinition`? and the same goes for `fields` property